### ISSUE TITLE
Unity Widgets not working

### ISSUE THREAD

**Issue by randoomx8** [sabresaurus/SabreCSG#244]
I have been following the learn page but I cant seem to move any brushes anymore.

After having created a small scene, brushes suddenly stopped responding to widgets, whenever I click and drag on the axsis widget to move the position of the brush, it instead behaves as if I was attempting to drag-select something on the screen.

**Comment by Henry00IS** [comment]
Hey randoomx8, that's very unfortunate! Could you please provide some further details:

What Unity version are you using?
Are there any error messages in the console?
Did you try pressing "Force Rebuild" in the toolbar?

**Comment by Henry00IS** [comment]
I have the feeling that perhaps you forgot to rebuild the world or disabled "Auto Rebuild" so the mesh you see isn't where the brushes are- so you end up starting a selection in the middle of nowhere causing the "drag-select".

**Comment by randoomx8** [comment]
Hey Henry! 
Im using the latest version of both unity (2019.2.15f1) and CSG (downloaded it yesterday from github).
There are no errors in the console, though every hour or so I would get at random a Win32 error with a massive log (searched it up but found nothing about it, it caused no issues so I just cleared it, it then disappears for about another hour)
I did indeed try Force Rebuild and it worked on two occasions, but it seems to not be a silver bullet; Auto rebuild showcased the same issues.

**All of the above said, today it seems like the classic "it suddenly fixes itself for no reason when IT is present" has happened.** Seems like all it needed was the PC to restart.
